cli/src/main/scala/com/ibm/sparktc/sparkbench/workload/ml/MLWorkloadResults.scala

```diff
@@ -0,0 +1,9 @@
+package com.ibm.sparktc.sparkbench.workload.ml
+
+case class MLWorkloadResults(
```

Comment:
Right you are.  I'm deleting it as well, and it should merge fine if we both delete.